examples/machinedeployment/machinedeployment.yaml

```diff
@@ -72,9 +72,8 @@ spec:
         owner: root
         path: /etc/kubernetes/cloud.conf
         permissions: "0600"
-      - content: |-
-          # cacert to communicate with OpenStack
-          $OPENSTACK_CLOUD_CACERT_CONFIG
+      - content: |
+          $OPENSTACK_CLOUD_CACERT_CONFIG6
```

Comment:
Did you try the machinedeployment? 6 spaces shouldn't be enough (I had to indent cloud provider config with 10)

Reply:
you are right, I forget the check this file after modification
10 used to work before my modification :(